SpatialGDK/Source/SpatialGDK/Private/Interop/ActorSystem.cpp

```diff
@@ -235,8 +215,28 @@ void ActorSystem::Advance()
 		}
 	}
 
-	CreateEntityHandler.ProcessOps(*SubView->GetViewDelta().WorkerMessages);
-	ClaimPartitionHandler.ProcessOps(*SubView->GetViewDelta().WorkerMessages);
+	for (const EntityDelta& Delta : TombstoneSubView->GetViewDelta().EntityDeltas)
+	{
+		if (Delta.Type == EntityDelta::ADD || Delta.Type == EntityDelta::TEMPORARILY_REMOVED)
+		{
+			AActor* EntityActor = TryGetActor(
+				UnrealMetadata(ActorSubView->GetView()[Delta.EntityId]
```

Comment:
Nit: we could add another version of `DeserializeComponent<T>` that takes in a `const FSubView&`.

Reply:
yeah, that would be nice- I wonder if it makes more sense for that function to take a view reference rather than a coordinator?

either way will probably leave this for now, unless you feel strongly. I think this is explicit in what it's doing even if it's a little wordy